scene/mission-replay/src/main/java/com/esri/samples/scene/AnimationModel.java

```diff
@@ -0,0 +1,71 @@
+package com.esri.samples.scene;
```

Comment:
For Animation, Camera, and Plane model all that needs to be done is make sure that all methods have a description. :frowning: